scripts/averaging/create_TEM_files.py

```diff
@@ -136,53 +174,75 @@ def create_TEM_files(adf):
         if not starting_location.is_dir():
             emsg = f"Provided 'cam_hist_loc' directory '{starting_location}' not found."
             emsg += " Script is ending here."
-            adf.end_diag_fail(emsg)
+            return
         #End if
 
         #Check if history files actually exist. If not then kill script:
         hist_str = '*.cam.'+hist_num
         if not list(starting_location.glob(hist_str+'.*.nc')):
             emsg = f"No CAM history {hist_str} files found in '{starting_location}'."
             emsg += " Script is ending here."
-            adf.end_diag_fail(emsg)
-        #End if
-
-        # open input files
-        shist = glob(f"{starting_location}/*{hist_num}.{start_year}*.nc")
-        ehist = glob(f"{starting_location}/*{hist_num}.{end_year}*.nc")
-        hist_files = sorted(shist + ehist)
-
-        ds = xr.open_mfdataset(hist_files)
-
-        #iterate over the times in a dataset
-        for idx,_ in enumerate(ds.time.values):
-            if idx == 0:
-                dstem0 = calc_tem(ds.squeeze().isel(time=idx))
-            else:
-                dstem = calc_tem(ds.squeeze().isel(time=idx))
-                dstem0 = xr.concat([dstem0, dstem],'time')
-            #End if
+            return
         #End if
 
-        #Update the attributes
-        dstem0.attrs = ds.attrs
-        dstem0.attrs['created'] = str(date.today())
-        dstem0['lev']=ds['lev']
+        #Get full path and file for file name
+        output_loc_idx = output_loc / case_name
 
-        output_loc_idx = Path(output_loc) / case_name
         #Check if re-gridded directory exists, and if not, then create it:
         if not output_loc_idx.is_dir():
             print(f"    {output_loc_idx} not found, making new directory")
             output_loc_idx.mkdir(parents=True)
         #End if
 
-        # write output to a netcdf file
-        dstem0.to_netcdf(output_loc_idx / f'{case_name}.TEMdiag_{start_year}-{end_year}.nc', 
-                            unlimited_dims='time', 
-                            mode = 'w' )
+        #Set baseline file name
```

Comment:
I think this comment should be:

```suggestion
        #Set case file name
```